components/menu/menu.css

```diff
@@ -0,0 +1,102 @@
+/* @import url('../../components/root.css'); */
+
+@import url('../../components/framer.css');
+
+
+
+
+.nav-dark{
+   background: var(--col-dark);
+}
+ 
+.z-index-max{
+   z-index: 9999;
+}
+
+.icon-nav-menu{
+   border-radius: 50%;
+   padding: 5px;
+   color: #fff;
+   cursor: pointer;
+}
+
+.icon-nav-menu:hover{
+   background: #999a9b;
+   transition: background 0.4s ease-in-out;
+}
+
+.sidebar-menu{
+   display: block;
+   position: fixed;
+   height: 100vh;
+   background-color: #333333;
+   width: 1px;
+   top:0;  
+   z-index: 1111;  
+   overflow-y: auto;
+}
+
+.sidebar-text{
+   font-size: 1.2rem;
+   color: #f9fafb;
+}
+.sidebar-text:hover{
+   color: #f9fafb;
+}
+.selected-text{
+   padding: 10px 0px 10px 20px;
+   color: red;
+}
+.selected-text:hover{
+   background: #636364;
+}
+
+.selected-text:focus{
+   background: #636364;
```

Comment:
Color variables here as well

Reply:
will make the changes thanks for pointing out